indra/newview/llagentcamera.h

```diff
@@ -112,6 +112,7 @@ class LLAgentCamera
 	//--------------------------------------------------------------------
 public:
 	void switchCameraPreset(ECameraPreset preset);
+	ECameraPreset getCameraPreset() const { return mCameraPreset; }
```

Comment:
Why is this needed?

Reply:
See the second file in this commit

Reply:
And what would happen with custom presets?

Reply:
I just want to mention that all 3 "default" presets (Front, Rear, Side) can be overwritten. For example, you can set the values of 'Rear' and 'Side' to be the same as 'Front'. Basically, they are almost the same as 'custom' presets, but have their own buttons and default values, stored in the file.

Reply:
> So in this case, it looks like those bugs will appear in the very first round of testing. Why wait, then?

Nothing is changed for custom presets
If we don't have bugs with them right now, why should they appear after this change?

Reply:
I meant something like that:
- Open 'Camer controls'
- Select 'Front view'
- Click 'Save as preset' button
- Choose 'Replace a preset' and select 'Rear'
- (then repeat and select 'Side')
- Click 'Save as preset' button once again
- And save as new preset.

After that the bug will be reproducible on all camera preset views, including REAR, GROUP, CUSTOM.

Reply:
Test plan for QA now includes testing presets, so I'm tentatively approving this.

Reply:
If the user explicitly sets incorrect preset settings, the bugs like this are expected, but they won't be treated as bugs

For example, if user choses the color scheme with white color for font and background, the text becomes unreadable, but this is not a buy, yes? :-)